content/en/docs/guides/Getting Started/Bringup/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Bringup Mardan Robot"
+linktitle: "Bringup"
+date: 2023-07-21
+weight: 2
+collapsible: false
+---
+
+{{< alert title="Bringup" >}}
+The code will be extracted from the [MardanRobotRpi](https://github.com/roboticamed/MardanRobotRpi) GitHub repository. 
+{{< /alert >}}
+
+### 1. Install Docker Compose  into RPi 4
+``` shell
+sudo apt-get install -y libffi-dev libssl-dev python3-dev python3 python3-pip
+
+sudo pip3 install docker-compose
+
+sudo systemctl enable docker
```

Comment:
Isn't docker already enabled from the rpi_setup document? If not, we should add these instructions there.

Reply:
solved